sdk/core/core/test/cmocka/CMakeLists.txt

```diff
@@ -20,23 +20,14 @@ endif()
 add_cmocka_test(${TARGET_NAME} SOURCES
                 main.c
                 test_az_json.c
```

Comment:
Should we sort these, maybe alphabetically, for easier look up/search? Same with  the signatures listed in `az_test_definitions.h`?

Reply:
sure, that can be done